WOXML.js

```diff
@@ -0,0 +1,142 @@
+var moment = require("moment");
+var util = require("./util");
+var builder = require("xmlbuilder");
+
+module.exports = class WOXML {
+    constructor(data, agencyMap) {
+        this.root = builder.create("WONetworkDeal");
+        this.data = data;
+        this.agencyMap = agencyMap;
+    }
+
+    getXML() {
+        return this.root;
+    }
+
+    buildHeader() {
+        var header = this.root.ele("Header")
+            .ele("ChangeStatus", "New Deal").up()
+            .ele("Property", "AMC").up() //Network?
```

Comment:
is this a hardcoded value (as in it will always be AMC and no other network)? should this be passed with the data?